OracleManagementAgent/dockerfiles/latest/container-scripts/install_zip.sh

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) 2022 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Please remove "All rights reserved." as we no longer include this phrase.

Reply:
fixed